docs/en/week01/01-1.md

```diff
@@ -61,21 +61,21 @@ The more efficient way is called Stochastic Gradient Descent (SGD). Since we wan
 
 ### [Computing gradients by backpropagation](https://www.youtube.com/watch?v=0bMe_vCZo30&t=2336s)
 
-Computing gradients by backpropagation is a practical application of the chain rule. The backpropagation for state gradients is as follows:
+Computing gradients by backpropagation is a practical application of the chain rule. The backpropagation equation for the input gradients is as follows:
```

Comment:
@skyil7, see update to weird terminology.

Reply:
Just perfect.
I will update it in Korean.